GVFS/GVFS.Common/GVFSPlatform.cs

```diff
@@ -120,13 +120,31 @@ public string GVFSPostIndexChangedHookExecutableName
 
             public string MountExecutableName
             {
-                get { return "GVFS.Mount" + this.ExecutableExtension; }
+                get { return "gvfs.mount" + this.ExecutableExtension; }

```

Comment:
Why was this changed, and is the casing consistent between Mac and Windows?

Reply:
The case doesn't matter on Windows, so I just changed this to match the casing on Mac.

Reply:
Out of curiosity, how did the casing matter on Mac?  I thought it was also case insensitive.

Reply:
Hm, good question actually. I keep getting tripped up that bash is case sensitive, but you're right that in theory this shouldn't have mattered on Mac either. The reason I changed it is that the "under construction" code I ripped out of CloneVerb was just replacing "GVFS.Mount" with "gvfs.mount". Let me give it a try and see.

Reply:
Yep upper case worked fine. As you pointed out offline, that under construction flag was originally added back when we had to run `dotnet gvfs.mount.dll`, and it hasn't been needed since we started publishing an executable.